### PR TITLE
[DO NOT MERGE] PROD. Added Polygon Network config.

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: .env.prod [hunk 1]
```diff
@@ -22,6 +22,17 @@ REACT_APP_ETHEREUM_ERC20_TOKEN_ADDRESS="0x2da719db753dfa10a62e140f436e1d67f2ddb0
 REACT_APP_ETHEREUM_ERC20_NAME="CERE Network"
 REACT_APP_ETHEREUM_ERC20_SYMBOL="CERE"
 
+### Polygon
+REACT_APP_POLYGON_CHAIN_ID="137"
+REACT_APP_POLYGON_NETWORK_ID="137"
+REACT_APP_POLYGON_NETWORK_NAME="Polygon Mainnet"
+REACT_APP_POLYGON_BRIDGE_ADDRESS="update mainnet bridge smart contract address"
```
Comment (AndreiNavoichyk) [inline .env.prod]: Wrap with braces.

File: .env.prod [hunk 2]
```diff
@@ -22,6 +22,17 @@ REACT_APP_ETHEREUM_ERC20_TOKEN_ADDRESS="0x2da719db753dfa10a62e140f436e1d67f2ddb0
 REACT_APP_ETHEREUM_ERC20_NAME="CERE Network"
 REACT_APP_ETHEREUM_ERC20_SYMBOL="CERE"
 
+### Polygon
+REACT_APP_POLYGON_CHAIN_ID="137"
+REACT_APP_POLYGON_NETWORK_ID="137"
+REACT_APP_POLYGON_NETWORK_NAME="Polygon Mainnet"
+REACT_APP_POLYGON_BRIDGE_ADDRESS="update mainnet bridge smart contract address"
+REACT_APP_POLYGON_ERC20_HANDLER_ADDRESS="update mainnet ERC20 handler smart contract address"
+REACT_APP_POLYGON_RPC_URL=""
```
Comment (AndreiNavoichyk) [inline .env.prod]: Do we need this parameter at all?

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,3 +5,4 @@
 - Added home path to logo and title
 - Added polygon configuration to chainbridgeConfig
 - Updated title
+- Updated env for prod with Polygon value
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md]: `Updated env.prod with Polygon Network`.

Comment (karthikeyan-cere) [reply CHANGELOG.md]: Updated

File: .env.prod (hunk 1 above)
Comment (karthikeyan-cere) [reply .env.prod]: Updated.

File: .env.prod (hunk 2 above)
Comment (karthikeyan-cere) [reply .env.prod]: No, We can hard code few parameters like token name, symbol and RPC URL. Here is the ticket for [it](https://cerenetwork.atlassian.net/browse/CBI-1422).

**Pull request closed without merging**